src/commonMain/kotlin/com/algolia/search/endpoint/EndpointSearch.kt

```diff
@@ -122,25 +122,37 @@ public interface EndpointSearch {
         requestOptions: RequestOptions? = null
     ): ResponseSearch
 
+    @Deprecated(
+        message = "Use findObject instead.",
+        replaceWith = ReplaceWith("findObject(match, query, doNotPaginate, requestOptions)"),
```

Comment:
```suggestion
        replaceWith = ReplaceWith("findObject(match, query, paginate, requestOptions)"),
```

Reply:
Excellent catch 💯

Reply:
Spoke too fast, actually, it need stay as `doNotPaginate`, this allows the IDE to replace the old variable with the new one (So it needs to have the name of the old variable to perform the match properly)